### DISCUSSION TITLE
how i can add thumbnails to  image?

### DISCUSSION THREAD

**Discussion by BatuhanAkar** [Tyrrrz/YoutubeExplode#623] [Q&A]
i tried every solution I could find, tried a lot of ways on my own, but I couldn't view the thumbnails. how can I do that?

**Comment by Tyrrrz** [comment]
https://github.com/Tyrrrz/YoutubeExplode/blob/5ad0b15e07131d32d0ef40bfa7513a386a69e5fa/YoutubeExplode.DemoWpf/Views/MainWindow.xaml#L75-L76

https://github.com/Tyrrrz/YoutubeExplode/blob/5ad0b15e07131d32d0ef40bfa7513a386a69e5fa/YoutubeExplode.DemoWpf/ViewModels/MainViewModel.cs#L67-L76

**Reply by BatuhanAkar** [reply]
thanks, a few hours after creating the question, I found it as a result of my work, but I still wanted to thank you for your interest.
